### PR TITLE
WiX: package Observable into the distribution

### PR DESCRIPTION
**Pull request by compnerd**
Include the Observable module in the SDK and runtime.

### CODE REVIEW

File: platforms/Windows/rtl/lib/rtllib.wxs [hunk 1]
```diff
@@ -38,6 +38,9 @@
       <Component>
         <File Source="$(SDK_ROOT)\usr\bin\swiftCRT.dll" />
       </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\bin\swiftObservation.dll" />
```
Comment (phausler) [inline platforms/Windows/rtl/lib/rtllib.wxs:42]: do we need to also provide the macros package too?

**Review by phausler — COMMENTED** [review]
the library part seems reasonable location wise.

File: platforms/Windows/rtl/lib/rtllib.wxs (hunk 1 above)
Comment (compnerd) [reply platforms/Windows/rtl/lib/rtllib.wxs:42]: I didn't know about the macros package!  Could you point me to that?  We will want to do that, but I think that for now it doesn't matter as macros aren't available yet on Windows (but working towards it).

Comment (phausler) [reply platforms/Windows/rtl/lib/rtllib.wxs:42]: the macro is created from here: https://github.com/apple/swift/tree/main/lib/Macros/Sources/ObservationMacros

**Review by phausler — APPROVED** [review]
it looks like all the appropriate parts are included; I am not familiar with the wxs format so I can't comment on the soundness of that part.

**Comment by compnerd** [comment]
Tested at https://ci-external.swift.org/job/swift-PR-build-toolchain-windows/1018/

**Pull request merged**